### DISCUSSION TITLE
Wishlist: Show content ot "category" element

### DISCUSSION THREAD

**Discussion by zagor** [spacecowboy/Feeder#285] [Ideas]
Some feeds contain useful information in the "category" element. It would be nice to be able to view this element. 
To avoid cluttering the screen in feeds that don't use this element, perhaps it could be an option per feed.

**Comment by crock** [comment]
@zagor Can you provide a couple of feed urls of example websites/blogs that implement this "category" field?

**Reply by spacecowboy** [reply]
One concrete one: https://cornucopia.se/feed/atom/

so many that it's just spam

**Reply by zagor** [reply]
Yeah some feeds seem to over-use the concept. :disappointed: 
The feed I wanted it for is https://mediakollen.org/rss/bloggar/all/, it only contains a single category field for each item.
